source/reference/configuration-options.txt

```diff
@@ -3201,10 +3201,19 @@ Key Management Configuration Options
    authenticating the user. :binary:`~bin.mongod` or :binary:`~bin.mongos` does not check the remaining documents
    in the array.
    
-   If the given document does not match the provided authentication name, or
-   the transformation described by the document fails, :binary:`~bin.mongod` or :binary:`~bin.mongos` continues
-   through the list of documents to find additional matches. If no matches are
-   found in any document, :binary:`~bin.mongod` or :binary:`~bin.mongos` returns an error.
+   If the given document does not match the provided authentication
+   name, :binary:`~bin.mongod` or :binary:`~bin.mongos` continues
+   through the list of documents to find additional matches. If no
+   matches are found in any document, or the transformation the document
+   describes fails, :binary:`~bin.mongod` or :binary:`~bin.mongos`
+   returns an error. 
+
+   Starting in MongoDB 4.4, :binary:`~bin.mongod` or
+   :binary:`~bin.mongos` also returns an error if one of the
+   transformations can't be evaluated due to networking or
```

Comment:
nit - we try to avoid contractions. can't -> cannot

Reply:
updated